scss/components/_responsive-embed.scss

```diff
@@ -29,7 +28,10 @@ $responsive-embed-ratio-widescreen: 16 by 9 !default;
 
 /// Creates a responsive embed container.
 /// @param {List} $ratio [$responsive-embed-ratio] - Ratio to use for the container, formatted as `x by y`.
-@mixin responsive-embed($ratio: $responsive-embed-ratio) {
+@mixin responsive-embed($ratio: null) {
```

Comment:
Is it still stuff restricted to `responsive-embed` only ?
- If yes, you could name it `responsive-embed-ratio`.
- If not, you could name it `box-ratio` or (`{something}-ratio`) for a global ratio modifier utility.